tests/run-make/raw-dylib-stdcall-ordinal/rmake.rs

```diff
@@ -0,0 +1,40 @@
+// `raw-dylib` is a Windows-specific attribute which emits idata sections for the items in the
+// attached extern block,
+// so they may be linked against without linking against an import library.
+// To learn more, read https://github.com/rust-lang/rfcs/blob/master/text/2627-raw-dylib-kind.md
+// Almost identical to `raw-dylib-link-ordinal`, but with the addition of calling conventions,
+// such as stdcall.
+// See https://github.com/rust-lang/rust/pull/90782
+
+//@ only-x86_64
```

Comment:
Discussion: actually this is a bit tricky, the original test worked for *both* 32-bit **and** 64-bit windows?

Reply:
Suggestion: please leave a comment about leaving *why* 32 bit (the calling convention is specific to x86, not x86_64).